lib/imageDisplay.ml

```diff
@@ -122,12 +131,10 @@ let print_portrait conf base p =
     [Rem] : Ne pas utiliser en dehors de ce module.                           *)
 let print_source conf f =
   let fname = if f.[0] = '/' then String.sub f 1 (String.length f - 1) else f in
```

Comment:
Do you know why removing '/' is needed? this is probably buggy on windows

Reply:
I dont know why. I suspect it is a remnant of the protection against accessing files outside the image folder.
Could probably be removed!